### PR TITLE
Changes the HOC activity cron job to account for weighted sessions.

### PR DESCRIPTION
**Pull request by ashercodeorg**
In doing so, removes historical cruft along the way.

### CODE REVIEW

File: lib/analyze_hoc_activity_helper.rb [hunk 1]
```diff
@@ -27,71 +14,56 @@ def add_count_to_hash(hash, key, count)
...
+      # The session does not specify the session weight, default to one.
+      " 1)" +
+    " ) AS count"
+  from_where = "FROM hoc_activity" +
+    " WHERE (started_at >= '#{day}' AND started_at < '#{next_day}')" +
+      " OR (pixel_started_at >= '#{day}' AND pixel_started_at < '#{next_day}')"
+  finished_from_where = "FROM hoc_activity" +
+    " WHERE (finished_at >= '#{day}' AND finished_at < '#{next_day}')" +
+      " OR (pixel_finished_at >= '#{day}' AND pixel_finished_at < '#{next_day}')"
+
```
Comment (Hamms) [inline lib/analyze_hoc_activity_helper.rb]: nit: align the `" OR ...` lines with the `" WHERE ...` lines

**Comment by ashercodeorg** [comment]
@philbogle

**Comment by Hamms** [comment]
LGTM

**Comment by philbogle** [comment]
LGTM

**Pull request merged**